bundles/org.openhab.binding.evcc/README.md

```diff
@@ -42,8 +43,46 @@ Default value for _refreshInterval_ is 30 seconds.
 
 ## Thing(s) Configuration
 
-The Things don't have a configuration. They will be set up automatically when a bridge has been configured.
-You can add them via file configuration or in the openHAB UI.
+Things will be set up automatically when a bridge has been configured and will appear in your inbox.
+It is not possible to add things manually in the UI.
```

Comment:
Why not? I would expect someone to do this as what is the difference to declaring a thing in the things file?

Reply:
Yeah you are right, it is not possible since the things will not be listed in the UI. The user can only select the bridge. I can change this if it is wished.

Reply:
Yeah you are right, it is not possible since the things will not be listed in the UI. The user can only select the bridge. I can change this if it is wished.